src/main/java/edu/ucsb/ucsbcslas/repositories/TutorAssignmentRepository.java

```diff
@@ -11,7 +11,30 @@
 
 @Repository
 public interface TutorAssignmentRepository extends CrudRepository<TutorAssignment, Long> {
-  List<TutorAssignment> findAll();
-  List<TutorAssignment> findAllByCourse(Course course);
-  List<TutorAssignment> findAllByTutor(Tutor tutor);
-}
+    List<TutorAssignment> findAll();
+
+    List<TutorAssignment> findAllByCourse(Course course);
+
+    List<TutorAssignment> findAllByTutor(Tutor tutor);
+
+    List<TutorAssignment> findAllByCourseId(Long courseId);}
```

Comment:
A very minor comment, but for style and readability it is important to have the final } of a multi-line object on its own line so that it's easier to see where it ends.